libs/domains/service-helm/feature/src/lib/values-override-arguments-setting/values-override-arguments-setting.tsx

```diff
@@ -0,0 +1,214 @@
+import { type PropsWithChildren, useState } from 'react'
+import {
+  Controller,
+  type UseFieldArrayRemove,
+  type UseFormReturn,
+  useFieldArray,
+  useFormContext,
+} from 'react-hook-form'
+import {
+  Button,
+  CodeEditor,
+  EmptyState,
+  Heading,
+  Icon,
+  IconAwesomeEnum,
+  InputSelectSmall,
+  InputTextSmall,
+  Popover,
+  Section,
+} from '@qovery/shared/ui'
+
+export type ArgumentTypes = 'json' | 'generic' | 'string'
+
+export interface HelmValuesArgumentsData {
+  arguments: {
+    key: string
+    type: ArgumentTypes
+    value: string
+    json?: string
+  }[]
+}
+
+export interface ValuesOverrideArgumentsSettingProps extends PropsWithChildren {
+  methods: UseFormReturn<HelmValuesArgumentsData>
+  onSubmit: () => void
+}
+
+function Row({ index, remove }: { index: number; remove: UseFieldArrayRemove }) {
+  const { watch, control } = useFormContext()
+
+  const [openEditor, setOpenEditor] = useState(true)
+  const valueTypeJson = watch(`arguments.${index}.type`) === 'json'
+
+  return (
+    <li className="mb-3 last:mb-0">
```

Comment:
Remaining props should be spread on `li` to pass `key` and other props

Reply:
https://github.com/Qovery/console/pull/1061/commits/038e701b38c313396c3bbe5ef0f7da2fc410f311